src/Compilers/CSharp/Portable/Symbols/Source/SourceModuleSymbol.cs

```diff
@@ -268,6 +268,20 @@ internal override void ForceComplete(SourceLocation? locationOpt, Predicate<Symb
                         _state.SpinWaitComplete(CompletionPart.FinishValidatingReferencedAssemblies, cancellationToken);
                         break;
 
+                    case CompletionPart.StartInterceptorsChecks:
```

Comment:
>case CompletionPart.StartInterceptorsChecks: [](http://example.com/codeflow?start=20&length=44)

This additional work during ```ForceComplete``` feels completely unnecessary. I think that whatever we are trying to achieve by that can be achieved through alternative means without affecting batch compilation scenarios.